cmd/commands/application/command.go

```diff
@@ -66,6 +58,21 @@ func NewCommand() (*cli.App, error) {
 	return app, nil
 }
 
+func registerFlags(flags *[]cli.Flag) error {
+	if err := cmd.RegisterDirectoryFlags(flags, &options); err != nil {
+		return err
+	}
+
+	*flags = append(*flags, tequilapiAddressFlag, tequilapiPortFlag)
```

Comment:
I was talking about moving all `appends` together: 
```golang
 	cmd.RegisterNetworkFlags(flags, &options)

	*flags = append(*flags, tequilapiAddressFlag, tequilapiPortFlag)
 	*flags = append(*flags, openvpnBinaryFlag, ipifyUrlFlag, locationDatabaseFlag)
	*flags = append(*flags, cliFlag)
```

Reply:
But I want to keep specific order of flags

Reply:
oh, clear